ledger/participant-integration-api/src/docs/scala/com/daml/platform/docs/SelfServiceErrorCodes_MigrationGuideGen_App.scala

```diff
@@ -67,8 +67,8 @@ object SelfServiceErrorCodes_MigrationGuideGen_App {
       tableLinesWithoutUnchangedErrorCodes,
```

Comment:
What file naming convention does this tool follow? It doesn't seem to conform to what we're using in the SDK, and its package vs. its location are not aligned.

Reply:
CC @pbatko-da . It's a temp app used for generating the migration guide. It will be removed before the final release to 1.18